README.rst

```diff
@@ -70,9 +70,110 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Windows** users:
 
-Note: You need to have bower installed before you may install the python packages.
+   - Clone the **coala-html** repository
+
+     ::
+
+       $ git clone https://github.com/coala/coala-html.git
+
+   - Download and install the latest version of Node.js from `official website <https://nodejs.org/en/download/>`__.
+
+   - Change the directory to **coala-html** and run following command to install dependencies
+
+     ::
+
+       $ python setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip install coala-bears
+
+   - Install **bower** to build static web pages
+
+     ::
+
+       $ npm install -g bower
+
+
+   - Run **coala-html** to display results from **coala-json**
+
+     ::
+
+       $ coala-html --dir ..\Tests
+
+
+2. Installation instructions for **Ubuntu/Linux** users:
+
+   - Clone the **coala-html** repository
+
+     ::
+
+      $ git clone https://github.com/coala/coala-html.git
+
+   - Install latest version of **Node.js**
+
+     a. For **Ubuntu** users:
+
+        * *Configure Node.js PPA*
+
+          ::
+
+           $ sudo apt-get install curl python-software-properties
+           $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
+
+        * *Install Node.js and npm*
+
+          ::
+
+           $ sudo apt-get install nodejs
+
+     b. For installing Node.js via Package manager on other distributions, refer `official guide <https://nodejs.org/en/download/package-manager/>`__.
+
+   - *Check Node.js and npm version*
+
+     You can find more details about current version on node.js `official website <https://nodejs.org/>`__.
+
+     ::
+
+       $ node -v
+
+       $ npm -v
+
+   - Install **bower** to build static web pages
+
+     ::
+
+       $ sudo npm install -g bower
+
+   - Change the directory to **coala-html** and run following command to install dependencies
+
+     ::
+
+       $ python3 setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip3 install coala-bears
+
+   - Change ownership to install packages required by bower
+
+     ::
+
+       $ sudo chown -R $USER:$GROUP ~/.config/configstore
```

Comment:
Sorry if my review repeats things already in old reviews,...

This step is a non-standard step -- there is no reason why this directory should have strange permissions.

The problem is possibly caused by `sudo npm install -g bower`, but I am not sure about that.

Anyways ... you added it here for a reason ... i'd love to know why, so we can look at how to avoid it.

Reply:
Actually, when we try to run `coala-html --dir ./coala_test_results` it install the bower components that are required to build up a static website and it is a package used by AngularJS, that is possible only if it has write access to `configstore` 
and, to install `bower` using npm it needs write access that's why i used `sudo`

Reply:
Using asdf solves those sudo problems.

Reply:
Ok, will try out them and if everything goes well, will use asdf.

Reply:
@KVGarg , if you never run 'sudo' in your install steps, the `chown` is not necessary.

redo your steps without running 'sudo'.  and show me errors that you can not fix.  `chown` is not the solution - it is a workaround recommended by npm because they do not recommend `asdf`, and so their workaround is not acceptable to us.

Reply:
@jayvdb I'm wondering that earlier it wasn't getting installed due to write access to `configstore` but now it is getting installed easily without even using `sudo`. Removing this instruction from guide.